sphinx/source/docs/user_guide/geo.rst

```diff
@@ -33,7 +33,8 @@ Google Maps
 Bokeh can also plot glyphs over a Google Map using the :func:`~bokeh.plotting.gmap.gmap`
 function. You must pass this function `Google API Key`_ in order for it to work, as
```

Comment:
```suggestion
function. You must pass this function a `Google API Key`_ in order for it to work, as
```

Reply:
This was a previous typo but as long as these files are being edited it can be fixed at the same time.